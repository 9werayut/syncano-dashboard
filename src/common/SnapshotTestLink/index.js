import SnapshotTestLink from './SnapshotTestLink';

export default SnapshotTestLink;
