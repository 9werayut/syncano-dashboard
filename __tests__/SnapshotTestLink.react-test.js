import React from 'react';
import renderer from 'react-test-renderer';

import SnapshotTestLink from '../src/common/SnapshotTestLink';

it('renders correctly', () => {
  const tree = renderer.create(
    <SnapshotTestLink/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
