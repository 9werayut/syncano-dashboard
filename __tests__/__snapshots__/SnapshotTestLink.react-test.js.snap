exports[`test renders correctly 1`] = `
<a
  className="normal"
  href="#"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]} />
`;
